gp-nested-forms/gpnf-copy-parent-value-manually.js

```diff
@@ -25,7 +25,9 @@ gform.addAction( 'gpnf_init_nested_form', function( childFormId, gpnf ) {
 
 function copyParentFormValue( parentFormId, parentFieldId, childFormId, childFieldId ) {
 
-	var value = jQuery ( '#input_{0}_{1}'.gformFormat( parentFormId, parentFieldId ) ).val();
+	var value = jQuery ( '#input_{0}_{1}'.gformFormat( parentFormId, parentFieldId ) ).val() ||
+		jQuery('input[name="input_{0}"]:checked'.gformFormat( parentFieldId )).val() ||
+		jQuery('input[name="input_{0}"]:selected'.gformFormat( parentFieldId )).val();
```

Comment:
Just stating this for posterity, this whole `copyParentFormValue` is very basic and by the looks of it, would likely only work for copying a single value into a text/textarea input. I'm not sure how complicated we want to make this function.